readme.md

```diff
@@ -84,7 +84,7 @@ var result = Markdown.ToHtml("This is a text with some *emphasis*");
 Console.WriteLine(result);   // prints: <p>This is a text with some <em>emphasis</em></p>
 ```
 
-In order to activate most of all advanced extensions (except Emoji, SoftLine as HardLine, JiraLinks and SmartyPants)
+In order to activate most of all advanced extensions (except Emoji, SoftLine as HardLine, Bootstrap, YAML frontmatter, JiraLinks and SmartyPants)
```

Comment:
I would change the casing on frontmatter to FrontMatter (otherwise I always read it as "formatter").

LGTM otherwise, thank you